### PR TITLE
webapp descriptor

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: apps/webapp.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    runtime: "vm"
+    params:
+      image_hash: "c37c1364f637c199fe710ca62241ff486db92c875b786814c6030aa1"
+nodes:
+  db:
+    payload: "db"
+    entrypoint:
+      - ["/bin/run_rqlite.sh"]
+    network:
+      "default"
```
Comment (johny-b) [inline apps/webapp.yaml]: Why a newline here?

File: apps/webapp.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  http:
+    payload: "http"
+    entrypoint:
+      - ["/bin/bash", "-c", "cd /webapp && python app.py --db-address 192.168.0.2 --db-port 4001 initdb"]
+      - ["/bin/bash", "-c", "cd /webapp && python app.py --db-address 192.168.0.2 --db-port 4001 run > /webapp/out 2> /webapp/err &"]
+    http_proxy:
+      ports:
+        - "5000"  # specify just the remote port, allow the local port to be automatically chosen
+    network:
+      "default"
```
Comment (johny-b) [inline apps/webapp.yaml]: And here?

File: apps/webapp.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    ip:
+      - "192.168.0.3"
+    depends_on:
```
Comment (johny-b) [inline apps/webapp.yaml:29]: And here?

**Review by johny-b — APPROVED** [review]
LGTM, but I don't get the point of a few newlines, this looks inconsistent to me.

EDIT: I think we might want `depends_on` to be a list, now it's just a single value. But I'm not saying we must change this now.

File: apps/webapp.yaml (hunk 1 above)
Comment (shadeofblue) [reply apps/webapp.yaml]: mistake

File: apps/webapp.yaml (hunk 2 above)
Comment (shadeofblue) [reply apps/webapp.yaml]: same

File: apps/webapp.yaml (hunk 3 above)
Comment (shadeofblue) [reply apps/webapp.yaml:29]: and here it's a feature :) ... but updated according to our discussion :)

**Pull request merged**